schulcloud/LoginViewController.swift

```diff
@@ -25,6 +25,10 @@ class LoginViewController: UIViewController {
         usernameTextArea.delegate = self
         passwordTextArea.delegate = self
         usernameTextArea.text = defaults.string(forKey: usernameKey)
+        
+        DispatchQueue.global(qos: .default).sync {
+            recreatePersistentContainer()
+        }
```

Comment:
Still, I would prefer `recreatePersistentContainer` to be called at two different points rather than "saving" 3 lines of code. Additionally, `recreatePersistentContainer` is called twice on the initial app launch.